packages/rx_bloc_cli/mason_templates/bricks/feature_counter/__brick__/lib/l10n/network_lookup/counter_lookup.dart

```diff
@@ -1,15 +1,17 @@
 {{> licence.dart}}
 
 import '../../assets.dart';
```

Comment:
My understanding is that the network_loopkup files are not needed in case the remote translations are not enabled.

Reply:
My understanding is that we need the lookup only if the translations needs to be adjusted from the API. Is that correct statement ?